note-qualifying-protocols-and-encodings.md

```diff
@@ -0,0 +1,47 @@
+# Qualifying Protocols and Encodings
+
+The explicit goal of the CloudEvents effort, as expressed in the specification, is 
+"describing event data in a common way" and "to define interoperability of event 
+systems that allow services to produce or consume events, where the producer and 
+consumer can be developed and deployed independently".
```

Comment:
Technically you get this just using and agreed upon messaging queue. Cloud events is suppose to be for transportability for the workload so you can switch out the producer and not change your consumer code.